src/test/java/plugins/AnalysisCollectorPluginTest.java

```diff
@@ -329,6 +330,33 @@ public void should_aggregate_warnings_in_dashboard_portlet() {
         assertThat(dashboard, not(hasWarningsFor(job, TASKS, TASKS_ALL)));
     }
 
+    @Test
+    @WithPlugins({"git@2.4.0", "workflow-aggregator@1.10"})
```

Comment:
Well, it was an issue related to `analysis-test-plugin` dependency which was solved some time ago. Moreover this is a general test to check that the plugins works fine with Workflow so it is not trying to catch any concrete issue.